src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -44,6 +44,49 @@ const NodeWrapperPath = "/opt/node-wrapper/"
 const LocalIp = "0.0.0.0"
 const inspectParamVariableName = "ORYX_NODE_INSPECT_PARAM"
 
+// Checks if the application insights needs to be enabled for the current runtime
+func shouldApplicationInsightsBeConfigured() bool {
```

Comment:
@markwolff, for the node apps that have applicationinsights as dependency, should we assume they will have their own applicationinsightloader.js? or the loader will always be injected by oryx?

Reply:
Discussed/resolved this offline with Arjun. For apps which are deployed with app insights as a dependency, they will not provide their own loader. We should not start app insights for them because they are either doing it themselves or removed app insights (and forgot to remove the dependency)